### PR TITLE
Add educate/it to i18n sync

### PR DESCRIPTION
**Pull request by smusoke**
**What**: Add our `it.md` markdown file to our sync-in step and update the minimum requirements table to be more translatable.

**Why**: This will introduce the https://code.org/educate/it markdown page to our translation pipeline on the next sync. The table is now uses HTML to support more intricate styling([link to discussion](https://codedotorg.slack.com/archives/C0SUN2SSF/p1653410282123389)).

## Links

- jira ticket: [FND-1966](https://codedotorg.atlassian.net/browse/FND-1966)

## Testing story

Confirmed changes locally

Screen Shot 2022-05-27 at 11 21 41 AM

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/educate/it.md [hunk 1]
```diff
@@ -14,17 +14,57 @@ theme: responsive
...
+        </tr>
+        <tr>
+          <td>Mobile Safari 11.x</td>
+        </tr>
+        <tr>
+          <td>Edge 87.x</td>
+        </tr>
+        <tr>
+          <td>Firefox 91.x</td>
+        </tr>
```
Comment (daynew) [inline pegasus/sites.v3/code.org/public/educate/it.md:40]: Should we keep this part from being translatable by moving it to a `.partial`? Or using a `.erb` to dynamically populate the table?

File: pegasus/sites.v3/code.org/public/educate/it.md [hunk 2]
```diff
@@ -14,17 +14,57 @@ theme: responsive
...
+        <tr>
+          <td>Windows 7</td>
+        </tr>
+        <tr>
+          <td>Android 6</td>
+        </tr>
+        <tr>
+          <td>ChromeOS (Chromebooks)</td>
+        </tr>
```
Comment (daynew) [inline pegasus/sites.v3/code.org/public/educate/it.md:62]: Same comment

Comment (daynew) [inline pegasus/sites.v3/code.org/public/educate/it.md:62]: Same comment

**Review by daynew — CHANGES_REQUESTED** [review]

File: pegasus/sites.v3/code.org/public/educate/it.md (hunk 1 above)
Comment (smusoke) [reply pegasus/sites.v3/code.org/public/educate/it.md:40]: The plan at the moment is to hide these values in Crowdin so they aren't translated. Is there an advantage to using an `.erb`? I wasn't aware we have any existing dynamic values anywhere in regards to our minimum requirements

Comment (daynew) [reply pegasus/sites.v3/code.org/public/educate/it.md:40]: Oh I forgot that was the plan!

**Review by daynew — APPROVED** [review]

**Pull request merged**